packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -353,6 +362,10 @@ export class BlockNoteEditor<
           ...(this.filePanel ? [this.filePanel.plugin] : []),
           ...(this.tableHandles ? [this.tableHandles.plugin] : []),
           PlaceholderPlugin(this, newOptions.placeholders),
+          ...(this.options.animations ?? true
+            ? [PreviousBlockTypePlugin()]
+            : []),
+          NonEditableBlockPlugin(),
```

Comment:
I'm not sure `NonEditableBlockPlugin` should be under the `BlockNoteUIExtension` since it handles keystrokes for blocks without inline content. Let's maybe move it to `getBlockNoteExtensions` or back to the `blockContainer` node.